mesheryctl/internal/cli/root/system/model.go

```diff
@@ -413,6 +426,120 @@ mesheryctl exp model search [query-text]
 	},
 }
 
+// add command to push aritfacts to the oci registry
+var pushModelCmd = &cobra.Command{
+	Use:   "push",
+	Short: "push model",
+	Long:  "push a model to the registry",
+	Example: `
+// Push a model
+mesheryctl exp model push [path-to-model] --username [username] --password [password] --registry [registry] --tag [tag] --repository [repository]
+	`,
+	// skip preRunE as it is not required for this command
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return cmd.Help()
+		}
+
+		// path to model
+		modelPath := args[0]
+
+		// Push logic
+		fs, err := file.New(".")
```

Comment:
What's the location of the created file? The current working directory?
`use os.TempDir() or ~/.meshery`